content/content-tool/tool/src/webapp/vm/content/sakai_resources_list.vm

```diff
@@ -809,13 +809,34 @@
 			});
 		});
 
-		function openCopyrightWindow(theURL,winName,winSettings) {
-			jQuery.get(theURL, function(data){
-				jQuery('#dialog-copyright').html(data);
-				jQuery('#dialog-copyright').dialog('open');
-				}, "html");
-			return false;
-		}
+		function openCopyrightWindow(url,winName,winSettings) {
+			//Idea from http://stackoverflow.com/q/16086162/3708872
+			jQuery.ajax({
+				type: "GET",
+				url: url,
+				dataType: "html",
+				success: function(response, status, request) {
+					var disp = request.getResponseHeader('Content-Disposition');
+					//Might just want to do attachments, but for now put everything in a new window
+					if (disp) {
+						var form = jQuery('<form method="GET" action="' + url + '" target="_blank">');
+						jQuery('body').append(form);
+						form.submit();
+					} 
+					else {
+						jQuery.get(url, function(data, textStatus, jqXHR){
+							jQuery('#dialog-copyright').html(data);
+							jQuery('#dialog-copyright').dialog('open');
+						}, "html");
+					}
+
+					return false;
+				},
+			error: function (xhr, ajaxOptions, thrownError) {
+				//debugger;

```

Comment:
You mean something like 
window.console && console.log("Error from GET request"+thrownError);

Yeah that's probably worth it. I had a few problems where i didn't specify HTML and jQuery was expecting JSON and the code wasn't running at all. Otherwise it seemed fine.